foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/MicroserviceConfigLoader.java

```diff
@@ -69,8 +70,8 @@ public void loadAndSort() {
       }
 
       sort();
-    } catch (IOException e) {
-      throw new ServiceCombException("Failed to load microservice config", e);
+    } catch (Exception e) {
+      throw new ServiceCombException("Failed to load microservice configFile " + configFileFromClasspath + " and " + e.getMessage());
```

Comment:
throw new ServiceCombException("Failed to load microservice configFile " + configFileFromClasspath, e);

Reply:
done